### ISSUE TITLE
rereading of already opened log file 

### ISSUE THREAD

**Issue by karbofos123** [esrlabs/chipmunk#1022]
**Prerequisites**

affected version:

make sure to check this before filing an issue:

- [yes ] I checked the documentation and found no answer
- [ yes] I checked to make sure that this issue has not already been filed

**Describe what you would like to have**
Rereading of the same (already opened) file in some time intervals or continuously.   
A lot of times the log files are created constantly and for analysing of log files in real time it is necessary to see new data in files. 

**Potential Workaround**
Have no idea. 

**More Information**
Rereading can be done by keyboard shortcuts (for Example Ctr+R) or via Menu in the application.
Moreover, Auto rereading in defined by user time intervals (every 30s...1 min...5 min) could be useful feature for users.

**Comment by DmitryAstafyev** [comment]
@marcmo it looks like tail feature. Does it make sense to implement it now as soon as we have #1012 in work? I guess it will go there.

**Comment by DmitryAstafyev** [comment]
@karbofos123 btw thanks for your issue.

**Comment by marcmo** [comment]
@DmitryAstafyev you are right. We can implement it after #1012 is merged.
will close this here since it is a duplicate of #790
